corehq/apps/data_interfaces/templates/data_interfaces/interfaces/case_management.html

```diff
@@ -80,17 +80,16 @@
     <div id="case-management">
         <div class="row">
             <form class="well form-inline" style="margin: 1em; display: none;" data-bind="submit: updateCaseOwners, caseReassignmentForm: selected_cases">
-                    <label for="reassign_owner_select" class="inline">{% trans 'Reassign selected cases to' %} </label>
-                    <span data-bind="visible: should_show_owners">
-                        <input type="hidden" name="reassign_owner"
-                            id="reassign_owner_select"
-                            style="width: 300px;"
-                            data-bind="grabUniqueDefault: selected_owners,
-                                       optionsText: 'name',
-                                       optionsValue: 'ownerid',
-                                       optionsCaption: '{% trans 'Select New Owner...' %}',
-                                       event: {change: changeNewOwner}, ic"></select>
-                    </span>
+                <label for="reassign_owner_select" class="inline">{% trans 'Reassign selected cases to' %} </label>
+                <span data-bind="visible: should_show_owners">
+                    <input type="hidden" name="reassign_owner"
+                        id="reassign_owner_select"
+                        style="width: 300px;"
+                        data-bind="optionsText: 'name',
+                                   optionsValue: 'ownerid',
+                                   optionsCaption: '{% trans 'Select New Owner...' %}',
+                                   event: {change: changeNewOwner}, ic" />
```

Comment:
what is the `, ic` ?

Reply:
No clue, just changed the indentation here.

removed 338b716